source/contracts/trading/FillOrder.sol

```diff
@@ -97,7 +97,7 @@ library Trade {
         }
 
         // sell complete sets
-        uint256 _settlementFees = _data.contracts.completeSets.sellCompleteSets(this, _data.contracts.market, _numberOfCompleteSets);
+        var (_marketCreatorFees, _reporterFees) = _data.contracts.completeSets.sellCompleteSets(this, _data.contracts.market, _numberOfCompleteSets);
```

Comment:
I believe you can do `uint256 (_marketCreatorFees, _reporterFees) = ...` here.  `var` is only necessary (I think) if the types differ and you don't want to declare on a separate line.